BEP191.md

```diff
@@ -0,0 +1,204 @@
+<pre>
+  BEP: 191
+  Title: Validator Slash Mechanism 2.0
+  Status: Draft
+  Type: Standards
+  Created: 2022-01-17
+</pre>
+
+# BEP-191: Validator Slash Mechanism 2.0
+
+- [BEP-191: Validator Slash Mechanism 2.0](#bep-191-validator-slash-mechanism-20)
+   * [1. Summary](#1-summary)
+   * [2. Abstract](#2-abstract)
+   * [3. Motivation](#3-motivation)
+   * [4. Specification](#4-specification)
+      + [4.1 Block Difficulty](#41-block-difficulty)
+      + [4.2 New Punishment Rules](#42-new-punishment-rules)
+      + [4.3 Missing Block](#43-missing-block)
+      + [4.4 Delay Broadcast](#44-delay-broadcast)
+      + [4.5 Duplicated Block](#45-duplicated-block)
+      + [4.6 Persistence](#46-persistence)
+   * [5. Rational](#5-rational)
+   * [6. Backward Compatibility](#6-backward-compatibility)
+   * [7. License](#7-license)
+
+## 1. Summary
+
+This BEP introduces a validator slash mechanism 2.0 on the BNB Smart Chain.
+
+## 2. Abstract
+
+This BEP describes a new slash mechanism to slash malicious validator in more scenarios, if some validator missing proposal, it could slash directly. Otherwise, validators could propose a slash to the malicious validator, and when the majority of validators agree on it, the slash will affect too.
+
+There are slash scenarios:
+
+1. The validator misses the `INITURN` block;
+2. The validator delay broadcast `INTURN` block too long;
+3. The validator proposes a duplicated block in the same block height with a different block hash;
+
+The new slash rules refer to the previous ones. There is a certain tolerance threshold, and the punishment will be executed if the threshold is exceeded. This BEP can also be used in other scenarios where validators are punished for malicious behavior.
```

Comment:
The `Abstract` is not quite clear about what `scenarios` the current mechanism can not cover.